gradle/dependencies.gradle

```diff
@@ -9,21 +9,21 @@ configurations.all {
 ext {
   versions = [
     // Check https://oss.jfrog.org/libs-snapshot/io/opentelemetry/ for latest snapshot version.
-    opentelemetry: '0.7.0-20200723.154929-27',
```

Comment:
I reformatted this file using IntelliJ, no difference.  only line #12 and #14 were updated.